app/views/submissions/index.json.jbuilder

```diff
@@ -1,4 +1,8 @@
 json.array!(@submissions) do |submission|
-  json.extract! submission, :id, :code, :compiler, :result, :score
+  if user_signed_in? && current_user.admin == true
```

Comment:
How about also includes code when current_user == submission's user?

Reply:
Since users don't see their codes in the submissions page, maybe not to include code in submissions.json is a good option?

Reply:
Actually I'm not sure what page is this jbuilder for. I have no idea about any page that admin can see everyone's code, while non-admin can't see {him/her}self's code.

Reply:
The `index.json.jbuilder` provides https://tioj.ck.tp.edu.tw/submissions.json, and yeah "admin can see everyone's code, while non-admin can't see {him/her}self's code" is strange. It should provides something like [https://tioj.ck.tp.edu.tw/submissions](https://tioj.ck.tp.edu.tw/submissions).

Reply:
I mean I'm not sure about what this page is for, sorry. Is there any other pages using this returned value or it just provides an API?
If we can modify this page freely, how about just not include code for all cases?

Reply:
I guess there is no other pages using this returned value, so we can feel free to modify it.
I've updated it.